Documentation/configure-kubectl.md

```diff
@@ -18,9 +25,11 @@ $ chmod +x kubectl
 $ mv kubectl /usr/local/bin/kubectl
 ```
 
-Configure your local Kubernetes client using the following commands:
+## Configure `kubectl`
+
+Configure `kubectl` to connect to the target cluster using the following commands, replacing several values as indicated:
 
-* Replace `${MASTER_HOST}`
+* Replace `${MASTER_HOST}` with the address or name of the controller address or name used in previous steps
 * Replace `${CA_CERT}` with the absolute path to the `ca.pem` created in previous steps
```

Comment:
Yes I agree that is probably worth being more explicit about the replacement vars. Also agree we'll undertake in another PR.

Note to self; there are potentially more than one tracks back to the replacement, depending on what guide track you enter this doc form.